src/MeasureUtil/MeasureUtil.spec.js

```diff
@@ -38,8 +38,8 @@ describe('MeasureUtil', () => {
         const expectedShortLength = MeasureUtil.getLength(shortLine, map);
         const expectedLongLength = MeasureUtil.getLength(longLine, map);
 
-        expect(expectedShortLength).toEqual(99.99999999669033);
-        expect(expectedLongLength).toEqual(100545.83533277796);
+        expect(expectedShortLength).toEqual(99.88824008937313);
```

Comment:
Why did these values change?

Reply:
Also, floats/doubles should not be checked for equality, but compared using a delta (`0.1 + 0.2 !== 0.3).

Reply:
The default sphere radius taken into account for length/area calculation in `ol` is set to [6371008.8](https://github.com/openlayers/openlayers/blob/v5.1.3/src/ol/sphere.js#L32) whereas the previous implementation in `ol-util` used the non-default value [6378137](https://github.com/terrestris/ol-util/blob/master/src/MeasureUtil/MeasureUtil.js#L24) (that is the value equal to the semi-major axis of the WGS84 ellipsoid).

If needed the value may be passed to the corresponding util methods, but in here I would like to make use of the same radius as used in ol per default.

Reply:
@hwbllmnn Yes, absolutely! Fixed (and will be fixed for all other occurrences in a separate PR).